crates/rnote-engine/src/pens/eraser.rs

```diff
@@ -20,15 +20,48 @@ pub enum EraserState {
     Down(Element),
 }
 
+#[derive(Debug, Default, Clone, Copy)]
+pub struct EraserMotion {
+    last_element: Option<Element>,
+    last_element_time: Option<Instant>,
```

Comment:
I wonder if it makes sense to have only one tuple of Element, Instant instead of two separate elements given both will be either None or Some.

Reply:
That would make sense, yeah